package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "solhint",
-  "version": "3.3.8",
```

Comment:
Based on the amount of changes I'd say this should increase the minor and not just the patch version. I think it should be 3.4.0